tests/core/consensus/test_weight_proof.py

```diff
@@ -306,11 +308,11 @@ async def test_weight_proof_extend_multiple_ses(self, default_1000_blocks):
     async def test_weight_proof_from_database(self):
         connection = await aiosqlite.connect("path to db")
         block_store: BlockStore = await BlockStore.create(connection)
-        sub_blocks, peak = await block_store.get_sub_block_records()
-        headers = await block_store.get_header_blocks_in_range(0, 100225)
```

Comment:
Why are we reducing this? It seems like this would make the test catch less issues

Reply:
this is just a harness for debugging, doesn't run with ci, works with the testnet database